custom/icds_reports/dashboard_utils.py

```diff
@@ -60,3 +61,208 @@ def _get_nav_metadatada():
             'image': static('icds_reports/mobile/images/bulb.png')
         },
     }
+
+
+def _get_nav_menu_items():
+    return {
+        'Maternal and Child Nutrition': [
+            {
+                'name': 'Prevalence of Underweight (Weight-for-Age)',
+                'route': 'maternal_and_child/underweight_children',
+                'children': False,
+                'featureFlagOnly': False,
+                'showInMobile': True,
+                'showInWeb': True
+            },
```

Comment:
What do you think about using `@attr.s` for these? Then we can set default values and only override them when meaningful, which should substantially reduce the overall amount of boilerplate here.

Reply:
(another similar option would be `jsonobject`)

Reply:
http://www.attrs.org/en/stable/examples.html#converting-to-collections-types